app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java

```diff
@@ -0,0 +1,53 @@
+package org.fossasia.openevent.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.adapters.ScheduleListAdapter;
+import org.fossasia.openevent.data.Event;
+
+import java.util.ArrayList;
+
+/**
+ * Created by md91829 on 3/3/16.
+ */
+public class ScheduleFragment extends Fragment {
+
+    private RecyclerView recyclerView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'recyclerView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406791428/issues/source?bid=3233850&fileBranchId=3233907#l23)